addons/website_mass_mailing/static/tests/tours/newsletter_popup.js

```diff
@@ -21,7 +21,7 @@ tour.register('newsletter_popup_edition', {
     ...wTourUtils.clickOnSave(),
     {
         content: "Check the modal has been saved, closed",
-        trigger: '.o_newsletter_popup',
+        trigger: 'body:has(.o_newsletter_popup)',
         extra_trigger: 'body:not(.editor_enable)',
         run: function (actions) {
             const $modal = this.$anchor.find('.modal');
```

Comment:
Seems scary to change this without adapting the `run` method (this.$anchor is now different and would find all modal even invisible ones).

Reply:
Correct, I adapted it.